docs/feature-flag/feature_flag

```diff
@@ -0,0 +1,27 @@
+# How to Add a Feature Flag on Firebase
```

Comment:
Add the _md_ file extension so GH can render it correctly.

Reply:
Done.